### ISSUE TITLE
Control Node sends wrong events

### ISSUE THREAD

**Issue by drtwisted** [godotengine/godot#9981]
**Operating system or device - Godot version:**
(Android KitKat, arm) Godot 2.1.3 / Godot 2.1.4.official.beta

**Issue description:**
On Android control nodes interpret screen touches as MOUSE_BUTTON and MOUSE_MOTION events instead of SCREEN_TOUCH.

**Steps to reproduce:**
1. Create a simple scene with Control Node (checked with Label as well, probably should work for any node inherited from Control)
2. Attach a script
3. Connect the input_event signal
4. Fill in the test method body:
```
func _on_Control_input_event(event):
	print(str('Event type: ', event.type, ' is ', '' if event.is_pressed() else 'not', ' pressed'))
	if event.type == InputEvent.SCREEN_TOUCH and event.is_pressed():
		print('Screen touch!')
```

**Expected result:**
When you touch the Control area on a mobile device
 The output is:
```
Event type: 6 is pressed
Screen touch!
Event type: 6 is not pressed
```

**Actual result:**
When you touch the Control area on a mobile device
 The output is:
```
Event type: 3 is pressed
Event type: 2 is not pressed
Event type: 2 is not pressed
Event type: 2 is not pressed
Event type: 2 is not pressed
Event type: 3 is not pressed
```

For quick reference:
MOUSE_MOTION = 2
MOUSE_BUTTON = 3
SCREEN_TOUCH = 6

**Comment by eon-s** [comment]
Tested with touch emulation (because remote debug is not working for me), `Control._input_event` do not register touch events, directly or the signal way.

But `_input` does detect the touch.
Also `CollisionObject._input_event` detects touch too.

**Comment by akien-mga** [comment]
Fixed by #10136.
